apiary.apib

```diff
@@ -3261,6 +3261,10 @@ Get a list of tasks.
                 + completed: false (boolean)
                 + completed_at: `2016-02-04T16:44:33+00:00` (string)
                 + due_at: `2016-02-04T16:44:33+00:00` (string)
+                + scheduled_at: (array)
```

Comment:
I'm not sure about the naming here. The `_at` part maybe implies it's a datetime when in fact it's an object with two datetimes. I couldn't come up with a better name, suggestions very welcome.